google-cloud-tools-plugin/resources/META-INF/plugin.xml

```diff
@@ -28,6 +28,8 @@
         <li>Running and debugging App Engine standard applications on a local development server.</li>
         <li>Adding Google Cloud Java client libraries to your project.</li>
       </ul>
+      <p>These features are all available in IntelliJ IDEA. As of version 18.5.1, some of these features are available in other JetBrains IDEs, beginning with PyCharm.</p>
```

Comment:
instead of "some of these features", can you just say "Cloud Storage and Cloud Source Repositories integration"

I'm also one the fence about the "beginning with PyCharm" wording. Maybe just leave this part out. That way when we expand support for the other IDEs we won't need to re-release (and can just update the docs / feature matrix that you link to)

Reply:
this is optional, but while you are in the area, wdyt about updating the other wording to remove the "Google" prefix from the various products as appropriate?

Reply:
Good idea on making additional changes. Will do.

I hear you about removing the "beginning with PyCharm". That makes sense. In the same vein, if we list "Cloud Storage and Cloud Source Repositories integration", we will need to update this page when we expand the feature list. So, perhaps we should keep it general and rely on users to go to the feature matrix to see what features are available for their IDE of choice. WDYT?

Reply:
Adding more features for the other IDEs will require another release, so we can remember to update this description alongside that. So I’m ok leasing the actual features supported by PyCharm.

Reply:
Adding more features for the other IDEs will require another release, so we can remember to update this description alongside that. So I’m ok leasing the actual features supported by PyCharm.